win32ss/user/rtl/image.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * PROJECT:   ReactOS RTL
+ * LICENSE:   LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:   RtlGetExpWinVer function
+ * COPYRIGHT: Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#ifdef _WIN32K_
+    #include <win32k.h>
+
+    #define NDEBUG
+    #include <debug.h>
+#else
+    #include <user32.h>
+    #include <wine/debug.h>
+
+    WINE_DEFAULT_DEBUG_CHANNEL(user32);
+#endif
+
+/* Get the expected OS version from the application module */
+ULONG
+NTAPI
+RtlGetExpWinVer(_In_ PVOID BaseAddress)
+{
+    ULONG dwMajorVersion = 3, dwMinorVersion = 10; /* Set default to Windows 3.10 */
```

Comment:
Note: this corresponds with [WINVER_WIN31](https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/window.h;hb=172d65327b2c4e64f8fb9b5e9a02aa17b2be5a61#l58)

Reply:
Comment added.